.gitattributes

```diff
@@ -1,5 +1,7 @@
 * text=auto
 
+*.bat text eol=crlf
+*.patch -text
```

Comment:
I thought it would prevent the automatic updater to modify my patches:
https://github.com/conda-forge/pyside-feedstock/pull/20